ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -476,6 +481,62 @@ ORK_CLASS_AVAILABLE
 
 
 /**
+ The `ORKTextScaleAnswerFormat `class represents an answer format that includes a slider control with text choices.
+ 
+ The scale answer format produces an `ORKScaleQuestionResult` object that contains a number whose value is the selected slider value.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKTextScaleAnswerFormat : ORKAnswerFormat
+
```

Comment:
I'd add `- (instancetype)init NS_UNAVAILABLE;` here, and then I'd implement it as:

    - (instancetype)init {
        ORKThrowMethodUnavailableException();
        return nil;
    }